frontend/src/lib/lemon-ui/icons/icons.tsx

```diff
@@ -2427,6 +2427,18 @@ export function IconBugShield(props: LemonIconProps): JSX.Element {
     )
 }
 
+/** https://pictogrammers.com/library/mdi/icon/cursor-default-click/ */
+export function IconClick(props: LemonIconProps): JSX.Element {
```

Comment:
We should add this to the @posthog/icons repo

Reply:
Can we do that as a follow-up?